icicle/backend/cpu/src/ring/cpu_balanced_decomposition.cpp

```diff
@@ -322,5 +206,4 @@ static eIcicleError cpu_recompose_from_balanced_digits_PolyRing(
   return eIcicleError::SUCCESS;
 }
 
-REGISTER_BALANCED_DECOMPOSITION_POLYRING_BACKEND(
```

Comment:
Should `REGISTER_BALANCED_DECOMPOSITION_POLYRING_BACKEND` be removed from `vec_ops_backend.h`?